codewof/codewof/content/en/diamond-pattern/test-case-3-code.txt

```diff
@@ -0,0 +1 @@
+triangle(11)
```

Comment:
This and test-case-4 should probably be `diamond`

Reply:
This question has been removed because it is much harder than the rest of the questions.